app/javascript/components/recipes/new/create-recipes.vue

```diff
@@ -70,32 +71,117 @@
         @click="create"
       />
     </div>
+
+    <base-modal
+      @ok="addIngredientstoRecipe"
+      @cancel="toggleAddIngredientModal"
+      v-if="showingAddIngredientModal"
+      :title="$t('msg.ingredients.add')"
+      :ok-button-label="$t('msg.add')"
+      :cancel-button-label="$t('msg.cancel')"
+    >
+      <div class="flex f-row w-full">
+        <div class="mr-10">
+          <search-ingredient-list 
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Trailing spaces not allowed                                                   no-trailing-spaces